lib/core/proxy.c

```diff
@@ -818,7 +818,7 @@ static h2o_httpclient_head_cb on_connect(h2o_httpclient_t *client, const char *e
 
     if (req->overrides != NULL) {
         use_proxy_protocol = req->overrides->use_proxy_protocol;
-        props->use_expect = req->overrides->proxy_use_expect;
+        props->use_expect = req->overrides->proxy_expect_mode == H2O_PROXY_EXPECT_ENABLED;
```

Comment:
Do we want to rename `use_expect` to `send_own_expect` or something? I ask this because we now have two ways of "using expect."

Reply:
https://github.com/h2o/h2o/pull/3419/commits/eeb8dd01ccb81011cc3c8c66153e41fef9206196